lib/spack/env/cc

```diff
@@ -175,15 +175,26 @@ while [ -n "$1" ]; do
             ;;
         -Wl,*)
             arg="${1#-Wl,}"
-            if [ -z "$arg" ]; then shift; arg="$1"; fi
```

Comment:
Seems like this line is removed here but not for `-Xlinker`.  I think this makes sense and is actually a bugfix b/c a bare `-Wl,` shouldn't pass the next arg to the linker.  Nice catch.